Changes

```diff
@@ -73,7 +73,7 @@ In L<GH 192: nytprofhtml - internal anchor links in html pages go to wrong
 locations|https://github.com/timbunce/devel-nytprof/issues/192> Shawn Laffan
 reported that internal links to anchors in the generated HTML pages were not
 correctly positioned in the vertical dimension.   Discontinuing use of this
-Javascript file appears to correct the problem.
+JavaScript file appears to correct the problem.
```

Comment:
Some projects don't like changing changelogs. I'm happy to drop this or anything else.